test/srt/sampling/penaltylib/penalizers/test_min_new_tokens.py

```diff
@@ -0,0 +1,153 @@
+import typing
+import unittest
+
+import torch
+
+from sglang.srt.sampling.penaltylib.penalizers.min_new_tokens import (
+    BatchedMinNewTokensPenalizer,
+)
+
+from ..utils import (
```

Comment:
I prefer not to use relative imports. In sglang, we avoid them by moving common parts to `python/sglang/test/`